src/api/seeders/20190703095400-reasons.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  up: queryInterface => queryInterface.bulkInsert('Reasons',
+    [
+      {
+        description: 'VIOLENT',
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+      {
+        description: 'RAPE',
```

Comment:
@MCFrank16 these reasons should be descriptive text. 
So, replace `VIOLENT` with `It contains violent content`. 
Replace `HARASSMENT` with `This content is being used for harassment`

Apply this feedback to all the other descriptions in the seed.

Also, not sure why `RAPE` is here? I think it should be removed.

Reply:
yeah, lemme make it descriptive